projects/wave-core-new/src/lib/operators/dialogs/raster-type-conversion/raster-type-conversion.component.ts

```diff
@@ -0,0 +1,97 @@
+import {RasterLayer} from '../../../layers/layer.model';
+import {ResultTypes} from '../../result-type.model';
+import {AfterViewInit, ChangeDetectionStrategy, Component, OnDestroy, OnInit} from '@angular/core';
+import {FormControl, UntypedFormBuilder, UntypedFormGroup, Validators} from '@angular/forms';
```

Comment:
When you create a new operator, please use typed forms to not add more technical debts.

Reply:
[04e7ba2](https://github.com/geo-engine/geoengine-ui/pull/424/commits/04e7ba211b8feffab53221b5fa3149d7db26b67f)